app/controllers/clients_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ClientsController < ApplicationController
+
+  # GET /
+  def index
+  end
+
+  # GET /client
+  def show
+  end
+
+  def create
+    client = Client.new client_params
+    client.save!
```

Comment:
If this save errors we need to send back a json object with the error so it can be correctly handled on the front end.  As is I think the response envelope for the error message will be different from that of the success message.